### PR TITLE
created navbar

### PR DESCRIPTION
**Pull request by NehaChandra22**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -8,4 +8,186 @@
...
+}
+
+.nav-items a:hover {
+  /* color:#fff800; */
+  background-color: var(--light-purple-color);
+  border-radius: 10px;
+  padding: 5px;
+}
+.git:hover {
+  /* color: #fff800; */
```
Comment (NishantKumarTiwari123) [inline style.css:84]: remove commented codes

File: style.css [hunk 2]
```diff
@@ -8,4 +8,186 @@
...
+  .nav-items {
+    display: none;
+  }
+  .menu {
+    display: block;
+    color: var(--white--color);
+    font-size: 1.6rem;
+    padding: 5px;
+  }
+  #cross {
```
Comment (NishantKumarTiwari123) [inline style.css:132]: give more generic names so that reader can understand what this class is about

File: style.css [hunk 3]
```diff
@@ -8,4 +8,186 @@
...
+    padding: 30px;
+    gap: 20px;
+  }
+  /* media-nav-end- */
+}
+
+/* tab view */
+@media screen and (max-width: 998px) {
+  /* navbar */
+  nav {
```
Comment (NishantKumarTiwari123) [inline style.css:155]: Don't directly select elements like this, use classnames

**Review by NishantKumarTiwari123 — COMMENTED** [review]
Use any formatter before pushing your code, also put screenshots on PR description

File: script.js [hunk 4]
```diff
@@ -1 +1,15 @@
- 
\ No newline at end of file
+const sidebar = document.querySelector('.sidebar');
+const hidehamburger = document.querySelector('#hamburger');
+const hidecross = document.querySelector('#cross');
+console.log(sidebar);
```
Comment (saurabhmaurya45) [inline script.js:4]: remove console.log if there is no use. Or if it is for testing purpose only.

File: style.css (hunk 2 above)
Comment (NehaChandra22) [reply style.css:132]: changed in new branch "home-content".

File: style.css (hunk 3 above)
Comment (NehaChandra22) [reply style.css:155]: nav is a semantic tag , self-explanatory tag.

File: script.js (hunk 4 above)
Comment (NehaChandra22) [reply script.js:4]: yes,it was for testing purpose, removed now.

**Pull request merged**